src/fights.ts

```diff
@@ -385,6 +385,12 @@ function embezzlerSetup() {
     }
   });
   if (have($item`License to Chill`) && !get("_licenseToChillUsed")) use($item`License to Chill`);
+  if (globalOptions.ascending && get("questM16Temple") === "finished" && get("lastTempleAdventures") < get("knownAscensions")) {
```

Comment:
consider using questStep in lieu of === "finished", and myAscensions() in lieu of get("knownAscensions")